sonic-net/sonic-buildimage
# DellEMC: Fix API2.0 initialization issue

#### Why I did it
To fix https://github.com/sonic-net/sonic-mgmt/issues/8786

#### How I did it
Modified Fan API to check whether the data retrieved is valid or not and return accordingly

#### How to verify it
Verify whether API 2.0 is loaded properly or not.
Execute CLI's like "show version", "show interface status", "show platform psustatus" etc..

#### Which release branch to backport (provide reason below if selected)


#### Description for the changelog
[s5232f_UT.txt](https://github.com/sonic-net/sonic-buildimage/files/11915371/s5232f_UT.txt)
